03-seeking-help.Rmd

```diff
@@ -124,6 +124,11 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find out which function and with what parameters should you use
```

Comment:
Hmmm. I see what you're saying. It just feels a little off though. What about:
"Use help to find a function (and its associated parameters) that you could use to load data from a csv file in..."

Because you could use `read.csv(sep = "\t", ...)` or `read.table(sep = "\t", ...)` and both would work in this case.